script.js

```diff
@@ -1,52 +1,67 @@
 const formBtn = document.querySelector('#form-button');
 const booksList = document.querySelector('.books-list');
+let books = JSON.parse(localStorage.getItem('books'));
 
-let books = [];
+class Book {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
 
-const booksAdd = (id, title, author) => {
-  const li = document.createElement('li');
-  li.innerHTML = `<h2>${title}</h2>
-    <h3>${author}</h3>
-    <hr>`;
-  const deleteBtn = document.createElement('button');
-  deleteBtn.textContent = 'Remove';
-  li.insertBefore(deleteBtn, li.lastElementChild);
-  booksList.appendChild(li);
-  deleteBtn.addEventListener('click', () => {
-    books = books.filter((books) => {
-      if (books.title !== title && books.author !== author) {
+  addBook() {
```

Comment:
- [ ] [OPTIONAL] Even though this was not explicitly mentioned to you by the previous reviewer @rloterh , I strongly suggest you make this change as it was part of the project requirements. Kindly implement the local storage functionality when a user adds a book it persists in the browser's local storage. Likewise, when a user deletes a book it is also updated in the local storage.

Reply:
Okay, I will implement this change in my code before merging. Thanks for the review and approval